alimentador_principal/servo_bomba_agua/servo_bomba_agua.ino

```diff
@@ -41,7 +41,7 @@ int button3 = 0;
 Controller control(timing_array); FoodMachine foodMachine(myservo, 70, 110);
 
 void setup() {
-  control.setRepeat(2);//seting repeating time
+  control.setRepeat(3);//seting repeating time
```

Comment:
better to be able to set the repetion times as i can do it with the timings to activate the servo.